### ISSUE TITLE
Can you provide a runnable code?

### ISSUE THREAD

**Issue by qzhu2017** [sayred1/CNN#3]
@sayred1 
It looks like this repo is a bit messy. Can you clean it up and provide some instruction to run your code? I would like to know the exact progress and may help you at some point.

**Comment by sayred1** [comment]
@qzhu2017 Yes, I am currently rewriting part of the code since it was messy. Once I finish this section, I will upload it.

**Comment by sayred1** [comment]
@qzhu2017  Hey, sorry it took a while. I had to reformat the network since it was so messy. I have runnable code up right now, but there is still some debugging that I will do! I've also updated the read me file with instructions on how to run the code. If you have any questions, let me know.
